src/concrete/ml/quantization/quantized_ops.py

```diff
@@ -491,6 +491,68 @@ def copy_function(x):
             params=self.output_quant_params,
         )
 
+    def calibrate_analytical_output(self, *inputs: QuantizedArray) -> UniformQuantizer:
+        """Calibrate output quantization based on analytical formulas.
+
+        Args:
+            *inputs (QuantizedArray): quantized operation inputs. Quantized weights
+                are stored in the op instance
+
+        Returns:
+            res (UniformQuantizer): the quantizer of the operation's output values
+                that can be used to de-quantize these values.
+        """
+
+        q_input1 = inputs[0]
```

Comment:
implements the famous gemm calibration formula by identifying terms

Reply:
I am not sure I understand the whole analytical part on this PR. Why are we doing this?